x-pack/plugins/stack_connectors/README.md

```diff
@@ -219,15 +221,16 @@ The [Jira user documentation `params`](https://www.elastic.co/guide/en/kibana/ma
 
-| Property    | Description                                                                                             | Type                  |
-| ----------- | ------------------------------------------------------------------------------------------------------- | --------------------- |
-| summary     | The title of the issue.                                                                                 | string                |
-| description | The description of the issue.                                                                           | string _(optional)_   |
-| externalId  | The ID of the issue in Jira. If present, the incident is updated. Otherwise, a new incident is created. | string _(optional)_   |
-| issueType   | The ID of the issue type in Jira.                                                                       | string _(optional)_   |
-| priority    | The name of the priority in Jira. Example: `Medium`.                                                    | string _(optional)_   |
-| labels      | An array of labels. Labels cannot contain spaces.                                                       | string[] _(optional)_ |
-| parent      | The ID or key of the parent issue. Only for `Sub-task` issue types.                                     | string _(optional)_   |
+| Property    | Description                                                                                             | Type                                |
+| ----------- | ------------------------------------------------------------------------------------------------------- | ----------------------------------- |
+| summary     | The title of the issue.                                                                                 | string                              |
+| description | The description of the issue.                                                                           | string _(optional)_                 |
+| externalId  | The ID of the issue in Jira. If present, the incident is updated. Otherwise, a new incident is created. | string _(optional)_                 |
+| issueType   | The ID of the issue type in Jira.                                                                       | string _(optional)_                 |
+| priority    | The name of the priority in Jira. Example: `Medium`.                                                    | string _(optional)_                 |
+| labels      | An array of labels. Labels cannot contain spaces.                                                       | string[] _(optional)_               |
+| parent      | The ID or key of the parent issue. Only for `Sub-task` issue types.                                     | string _(optional)_                 |
+| otherFields | An object containing key-value pairs of any other fields in Jira without explicit properties.           | Record<string, string> _(optional)_ |
```

Comment:
Don't forget to change the doc from `Record<string, string>` to `Record<string, any>`.